railties/lib/rails/generators/rails/benchmark/benchmark_generator.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require "rails/generators/named_base"
+
+module Rails
+  module Generators
+    class BenchmarkGenerator < NamedBase
+      argument :reports, type: :array, default: []
+      class_option :method, type: :string, enum: %w(ips bm bmbm), default: "ips"
+
+      def generate_layout
+
+        if benchmark_method == "ips"
+          # TODO: Avoid having this run each time
```

Comment:
☝️ We should analyze the gemfile to see if IPS is already added, otherwise run this code and then add a test for it.